backend/src/polling_station/structs.rs

```diff
@@ -389,6 +397,15 @@ pub struct VotersCounts {
     pub total_admitted_voters_count: Count,
 }
 
+impl AddAssign<&VotersCounts> for VotersCounts {
+    fn add_assign(&mut self, other: &Self) {
+        self.poll_card_count += other.poll_card_count;
+        self.proxy_certificate_count += other.proxy_certificate_count;
+        self.voter_card_count += other.voter_card_count;
+        self.total_admitted_voters_count += other.total_admitted_voters_count;
+    }
+}
+
```

Comment:
There needs to be an implementation for adding votersRecounts to VotersCounts as well